src/org/thoughtcrime/securesms/WebViewActivity.java

```diff
@@ -41,15 +41,27 @@ protected void onCreate(Bundle state, boolean ready) {
 
     webView = findViewById(R.id.webview);
     webView.setWebViewClient(new WebViewClient() {
+      // `shouldOverrideUrlLoading()` is called when the user clicks a URL,
+      // returning `true` means, the URL is passed to `loadUrl()`, `false` aborts loading.
+      // the method is not called for POST request nor for on-page-links.
+      //
+      // nb: from API 24, `shouldOverrideUrlLoading(String)` deprecated and
```

Comment:
I've seen it multiple times now, and - what does nb mean :) ?

Reply:
https://en.wikipedia.org/wiki/Nota_bene I guess

Reply:
"nota bene", "by the way", draw attention to the reader to sth. else, https://en.wikipedia.org/wiki/Nota_bene

Reply:
Thanks :)